client/src/views/Dashboard/Dashboard.js

```diff
@@ -178,6 +178,13 @@ export default function Dashboard() {
     setTransactionSortDirection(direction)
   }
 
+  let claimPerDayLoadingIteration = 0
```

Comment:
Not a solution, as uses **side effect during rendering** 
Any helper functuion for format/transform data for jsx **should be pure** (for they could be easy cached if need via useMemo)